front-page.php

```diff
@@ -24,37 +24,50 @@
     if (!empty($works)) {
 ?>
 
-        <div id="home-work-holder" class="grid-row">
+        <div id="home-work-holder" class="grid-row padding-top-mid padding-bottom-large js-masonry margin-bottom-large">
 
 <?php
       foreach ($works as $work) {
         if (!empty($work['work_id'])) {
 
-          $work_image = get_the_post_thumbnail($work['work_id'], 'home-work', array('class'=>'home-work-image'));
+          $work_image = get_the_post_thumbnail($work['work_id'], 'full', false, array('data-no-lazysizes'=>true, 'class'=>'home-work-image'));
 
           if (!empty($work['image_id'])) {
-            $work_image = wp_get_attachment_image($work['image_id'], 'home-work', array('class'=>'home-work-image'));
+            $work_image = wp_get_attachment_image($work['image_id'], 'full', false, array('data-no-lazysizes'=>true, 'class'=>'home-work-image'));
           }
 
           $work_url = get_the_permalink($work['work_id']);
           $work_title = get_the_title($work['work_id']);
           $work_cats = get_the_category($work['work_id']);
 
+          $trans_y = (mt_rand(-10,10) / 10) * 50;
+          $trans_x = (mt_rand(-10,10) / 10) * 50;
+
+          $margin_top = mt_rand(1,4) * 50;
+          $margin_left = mt_rand(-2,2) * 50;
+          $margin = $margin_top . 'px 0 0 ' . $margin_left . 'px';
 ?>
 
-          <a class="home-work-item" href="<?php echo $work_url; ?>">
-            <?php echo $work_image; ?>
-            <div class="home-work-details grid-column text-align-center justify-around align-items-center">
-              <h2><?php echo $work_title; ?></h2>
-              <?php
-                if (!empty($work_cats)) {
-                  echo '<ul>';
-                  foreach ($work_cats as $cat) {
-                    echo '<li>' . $cat->name . '</li>';
-                  }
-                  echo '</ul>';
-                }
-              ?>
+          <a class="home-work-item grid-item item-s-12 item-m-6 item-l-4" href="<?php echo $work_url; ?>"
+            data-0-bottom-top="transform[sqrt]: translateY(<?php echo -($trans_y); ?>%) translateX(<?php echo -($trans_x); ?>%) scale(.7);"
```

Comment:
maybe not for this branch but could we put the scale value and maybe the random max mins as site settings?